### ISSUE TITLE
Add to MPC-HC Playlist does not work as expected in Windows 10 21H1

### ISSUE THREAD

**Issue by sKGu8gxV** [clsid2/mpc-hc#1327]
As Windows increasingly tries to mimic Apples "Closed garden" echo system so has programs abilities to change settings. One setting that I'm personally quite fond of is the ability to conveniently add files to my ever expanding playlist by right clicking a file and choosing "Add to mpc-hc playlist". Recently, after having freshly installed 21H1, when I do this I get "This file does not have an app associated with it for performing this action. Please install an app or, if one is already installed, create an assocation in the Default Apps Settings page."

How to reproduce this:

As the button in "Options > Formats" to "Set as default program" is greyed out I have "Open with" followed by opening the "Default Apps" app and choosing "Mpc-Hc" as the default video player. Having Mpc-hc show up there was not trivial in the first place. This changes the right click option for for example .mkv files from "Windows Media Player" to Mpc-hc. Mpc-hc works excellently except for this issue (And thanks a lot of keeping up the work clsid2!). 

Ps. I can do "Open With" but this clears my whole playlist. Not a desirable outcome.

**Comment by clsid2** [comment]
Click on the "Run as Administrator" button on Format options page. Then you can make the player create the file association registry keys, and let it register itself to be shown in "Default Apps".

Then step two is to go to Default Apps in Windows Settings and choosing MPC-HC as the actual default.

If you can't make that work, then I suggest installing MPC-HC as part of K-Lite Codec Pack. That will do it for you.

**Comment by sKGu8gxV** [comment]
> Then step two is to go to Default Apps in Windows Settings and choosing MPC-HC as the actual default.
 
> If you can't make that work, then I suggest installing MPC-HC as part of K-Lite Codec Pack. That will do it for you.

Yea I made it default in Default Apps. I can right click and choose "Add to MPC-HC Playlist". This is the error I get when I do that: https://i.imgur.com/hgSuQ52.png.

I checked in the extensions part of "Default Apps" app and it's correctly set. I have tried several different file extensions and get the same error. I will try K-lite, thanks for the tip.
